### PR TITLE
CONSOLE-2530: Migrate labels CRUD tests

### PR DESCRIPTION
**Pull request by rebeccaalpert**
Migrated labels CRUD tests from Protractor to Cypress.

Fixes https://issues.redhat.com/projects/CONSOLE/issues/CONSOLE-2530.

### CODE REVIEW

File: frontend/packages/integration-tests-cypress/tests/crud/labels.spec.ts [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+    cy.login();
+    cy.visit('/');
+    nav.sidenav.switcher.changePerspectiveTo('Administrator');
+    nav.sidenav.switcher.shouldHaveText('Administrator');
+    cy.createProject(testName);
+    cy.visit(`k8s/ns/${testName}/${plural}/~new`);
+    yamlEditor.isLoaded();
+    let newContent;
+    yamlEditor.getEditorContent().then((content) => {
+      newContent = _.defaultsDeep({}, yaml, safeLoad(content));
```
Comment (spadgett) [inline frontend/packages/integration-tests-cypress/tests/crud/labels.spec.ts:36]:
```suggestion
    yamlEditor.getEditorContent().then((content) => {
      const newContent = _.defaultsDeep({}, yaml, safeLoad(content));
```

File: frontend/packages/integration-tests-cypress/views/labels.ts [hunk 2]
```diff
@@ -0,0 +1,6 @@
+export const labels = {
+  inputLabel: (label: string) => cy.byTestID('tags-input').type(label),
+  confirmDetailsPageLabelExists: (label: string) => cy.get('.co-m-label__key').contains(label),
```
Comment (spadgett) [inline frontend/packages/integration-tests-cypress/views/labels.ts:3]: We should add a test ID for this

File: frontend/packages/integration-tests-cypress/views/labels.ts [hunk 3]
```diff
@@ -0,0 +1,6 @@
...
+  clickDetailsPageLabel: () => cy.get('.co-m-label__key').click(),
+  chipExists: () => cy.get('.pf-c-chip-group__label').should('exist'),
```
Comment (spadgett) [inline frontend/packages/integration-tests-cypress/views/labels.ts:5]: Ideally we'd add a test ID here as well if possible.

Comment (rebeccaalpert) [reply frontend/packages/integration-tests-cypress/views/labels.ts:5]: This one is a PF component and it doesn't look like I can pass one in.

**Comment by rebeccaalpert** [comment]
Addressed feedback and rebased.

File: frontend/packages/integration-tests-cypress/tests/crud/labels.spec.ts [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+import { errorMessage } from '../../views/form';
+import { modal } from '../../views/modal';
+import { labels } from '../../views/labels';
+
+describe('Editing labels', () => {
+  const name = `${testName}-editlabels`;
+  const plural = 'configmaps';
+  const kind = 'ConfigMap';
+  const labelValue = 'appblah';
+  const yaml = {
```
Comment (spadgett) [inline frontend/packages/integration-tests-cypress/tests/crud/labels.spec.ts:17]:
```suggestion
  const yaml: ConfigMapKind = {
```

File: frontend/packages/integration-tests-cypress/views/labels.ts (hunk 2 above)
Comment (spadgett) [reply frontend/packages/integration-tests-cypress/views/labels.ts:3]: Use the new test ID here?

**Comment by rebeccaalpert** [comment]
Changes are all set.

/test e2e-gcp-console /test analyze

**Review by spadgett — APPROVED** [review]
/lgtm

**Comment by spadgett** [comment]
/hold
Looks like the label tests are failing.

**Comment by rebeccaalpert** [comment]
Test failure seemed to be caused by usage of cy.get instead of cy.byTestID. Cypress won't run for me locally right now, so I'll double check when I can get it going again.

**Comment by rebeccaalpert** [comment]
Seems good now!

**Review by spadgett — APPROVED** [review]
/lgtm

**Comment by rebeccaalpert** [comment]
Removing the hold labels since this is now passing tests and doesn't require outside approvals.

**Pull request merged**